src/python/platforms/fuchsia/undercoat.py

```diff
@@ -188,5 +208,16 @@ def get_data(handle, fuzzer, src, dst):
   """Get files from a fuzzer on an instance, via undercoat.
   If src is a directory, it will be copied recursively. Standard globs are
   supported."""
-  return undercoat_instance_command('get_data', handle, '-fuzzer', fuzzer,
-                                    '-src', src, '-dst', dst).output
+  try:
+    return undercoat_instance_command(
+        'get_data',
+        handle,
+        '-fuzzer',
+        fuzzer,
+        '-src',
+        src,
+        '-dst',
+        dst,
+        abort_on_error=False).output
+  except UndercoatError:
+    return None
```

Comment:
we don't want to log here?

Reply:
The `logs.log_warn` in the inner `undercoat_api_command` will be triggered by the non-zero exit code, so anything here would be redundant. Sound good enough?

Reply:
Thanks, SGTM.